__tests__/app.test.js

```diff
@@ -127,29 +128,74 @@ describe("GET /api/articles", () => {
     });
 });
 
-describe("error tests", () => {
-    test("404: responds with 404 when non-existent path when endpoint is spelt incorrectly", () => {
+describe("GET /api/articles/:article_id/comments", () => {
+    test("status 200, responds with an array of comments with the needed properties", () => {
+        return request(app)
+        .get("/api/articles/3/comments")
+        .then(({ body }) => {
+            const expected = {
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                article_id: 3
+            }
+            expect(body.comments).toBeInstanceOf(Array);
+            expect(body.comments[0]).toEqual(expected);
+        });
+    });
+    test("responds with an array where comments are sorted by date in ascending order", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .then(({ body }) => {
+            expect(body.comments).toBeSortedBy("created_at");
+        });
+    });
+})
+
+describe("404 error tests", () => {
+    test("status 404: responds with 404 when endpoint is spelt incorrectly", () => {
         return request(app)
         .get("/api/tpics")
         .expect(404)
         .then((response) => {
             expect(response.body.msg).toBe("Endpoint Not Found");
         });
     });
-    test("404: responds with 404 when a valid id that is not currently in our data is used", () => {
+    test("status 404: GET/api/articles/:article_id: responds with 404 when a valid id that is not currently in our data is used", () => {
         return request(app)
         .get("/api/articles/30")
         .expect(404)
         .then((response) => {
             expect(response.body.msg).toBe("Endpoint Not Found");
         });
     });
-    test("400: responds with 400 when an invalid (incorrect data type) id is used", () => {
+    test("status 404: GET/api/articles/:article_id/comments: responds with 404 when a valid id that is not currently in our data is used", () => {
```

Comment:
It's a little confusing having your error handling for this endpoint mixed in with the error handling of other endpoints. I'd suggest having all of the tests, happy and sad path related to one endpoint together.